.github/ISSUE_TEMPLATE/documentation.yaml

```diff
@@ -0,0 +1,32 @@
+name: "📚 Documentation"
+description: "Report an issue related to documentation"
+title: "📚 Documentation: "
+labels: [documentation]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out our bug report form 🙏
```

Comment:
Please update this part as showed in our template. We cannot thank for bug report in documentation update.